runner/diff.go

```diff
@@ -63,3 +69,45 @@ func diffRunner(ctx context.Context, test *model.Test, sub model.KeyedSubmission
 	// TODO(flowlo): Save result.
 	return nil
 }
+func IODiffRun(ctx context.Context, params map[string]string, sub model.KeyedSubmission) error {
+	return errors.New("IO testing not yet available.")
+}
+
+func OutMatchDiffRun(ctx context.Context, params map[string]string, sub model.KeyedSubmission) (testResult model.DiffTestResult, err error) {
+	var str model.SimpleTestResult
+	str, err = Simple(ctx, sub)
+	if err != nil {
+		return
+	}
+	testResult = model.DiffTestResult{
+		SimpleTestResult: str,
+	}
+
+	var rc io.ReadCloser
+	rc, err = storage.NewReader(ctx, params["bucket"], params["tests"])
```

Comment:
"Normal input" would mean that the substring in `params["tests"]` is directly fed into the program? I'd rather skip this and have everything read from GCS.